tests/newsApi.test.js

```diff
@@ -0,0 +1,26 @@
+const NewsApi = require("../src/newsApi")
+require('jest-fetch-mock').enableMocks()
+
+describe("NewsApi", () => {
+	let subject;
+	beforeEach(() => {
+		subject = new NewsApi();
+	})
+
+	it("creates an instance of itself", ()=>{
+		expect(subject).toBeInstanceOf(NewsApi)
+	})
+
+	it("has a fetchNews() method", ()=>{
+		expect(typeof subject.fetchNews).toBe("function")
```

Comment:
Not seen this type of expect statement. I will use 'typeof' myself in the future.